frontend/public/components/overview.jsx

```diff
@@ -396,6 +408,28 @@ export class Overview extends React.Component {
       }
     ];
 
+    if (_.isEmpty(namespace) || namespace === ALL_NAMESPACES_KEY) {
+      return <div className="co-m-pane">
+        <Disabled>
+          <OverviewHeading disabled title={title} />
+        </Disabled>
+        <div className="co-m-pane__body">
+          <EmptyState>
```

Comment:
I'm hesitant to add two empty state styles in the console. I would prefer to use the existing console component and later convert them all to the Patternfly component if that's what we want.